### PR TITLE
upgraded babel to 7.x, updated readme

### PR DESCRIPTION
**Pull request by ppasmanik01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ The following examples show how to implement basic handler functions for consumi
 This example processes a DynamoDB Stream and publishes domain events to an EventBridge bus, which routes the events to the likes of a Kinesis stream. The details are explained below.
 
 ```javascript
-import { fromDynamodb } from 'aws-lambda-stream/from/dynamodb';
-import { publishToEventBridge as publish } from 'aws-lambda-stream/utils/eventbridge';
+import { fromDynamodb } from 'aws-lambda-stream/lib/from/dynamodb';
+import { publishToEventBridge as publish } from 'aws-lambda-stream/lib/utils/eventbridge';
```
Comment (jgilbert01) [inline README.md:28]: what change in the build can we add to eliminate the lib folder in package?

Comment (ppasmanik01) [reply README.md:28]: Normally, modules do not have path, so index should export all functions from all subdirectories that we want to expose.  The only other way is to use monorepo and build multiple modules like babel, or lodash, or middy.

Comment (jgilbert01) [reply README.md:28]: please combine this with the pr i just merged and then i'll merge this and publish a new version

Comment (ppasmanik01) [reply README.md:28]: ok

Comment (ppasmanik01) [reply README.md]: updated

**Comment by jgilbert01** [comment]
@ppasmanik01, i'll create a new branch and pr since I have made a bunch of other changes

**Pull request closed without merging**